src/big/big_decimal.cr

```diff
@@ -133,6 +134,13 @@ struct BigDecimal < Number
     initialize(num.to_s)
   end
 
+  # Creates a new `BigDecimal` from `BigRational`.
+  def initialize(num : BigRational)
```

Comment:
but why not make this a `self.new` then you can just return the original number instead of copying the value and scale...

Reply:
True indeed, I've forgot about that, thx! force-pushed again